src/router/handlers/resourceHandler.test.ts

```diff
@@ -365,7 +365,7 @@ describe('Testing search', () => {
         });
 
         // OPERATE
-        const searchResponse: any = await resourceHandler.typeSearch('Patient', { name: 'Henry' });
+        const searchResponse: any = await resourceHandler.typeSearch('Patient', { name: 'Henry' }, []);

```

Comment:
We should probably have tests with the allowedResourceType?

Reply:
yeah, this method is mostly a proxy to `searchService.typeSearch` which is mocked so the value of allowedResourceType doesn't really matter. 
I just added an assertion to verify that `allowedResourceType` is forwarded to `searchService.typeSearch`